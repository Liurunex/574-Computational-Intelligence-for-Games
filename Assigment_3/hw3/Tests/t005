#!/bin/bash
# Midfield

trap "/usr/bin/killall -q -u $USER TwoMinute 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15

/c/cs474/bin/run -stderr=/dev/null ./TwoMinute 2 11 45 60 < /dev/null

# clean up temp file
cd ..
rm -rf temp
