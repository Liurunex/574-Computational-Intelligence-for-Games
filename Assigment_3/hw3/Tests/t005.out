[0.000000, 0.066685, 0.933315]
[0.000000, 0.893249, 0.106751]
0.407154
